### DISCUSSION TITLE
Ctrl+Shift+L cannot be input as a shortcut.

### DISCUSSION THREAD

**Discussion by cpcopy** [microsoft/terminal#17564] [Q&A] [answered]
It can work on version 1.20.11381.0. But cannot work on version 1.20.11781.0. Is it a configuration issue or a bug no latest version?

**Comment by Yewzir** [comment]
@cpcopy 

You can reinstall version [1.20.11381.0](https://github.com/microsoft/terminal/releases/tag/v1.20.11381.0) from the official Microsoft Windows Terminal repository.

![image](https://github.com/user-attachments/assets/cb005b1e-e854-4057-a822-bd1bba094fd9)

This was noted about version 1.20.11781.0:

> This update was released to the Dev External and Canary channels on 2024-06-27, and if it behaves well under pressure it will be promoted to retail within a week.

I downgraded as well, for now.

Kind regards
Yewzir

**Reply by cpcopy** [reply]
Thanks for the suggestion. I downgraded the version. But the issue is still there. It looks like the issue only happens on my new win11 laptop.

**Comment by DHowett** [comment]
I can't reproduce this. Can you share some additional information about your system? What's your keyboard layout? Windows version? Are you using any global system hooks that override the keyboard? Have you installed something that bound <kbd>Ctrl+Shift+L</kbd> globally?

![image](https://github.com/user-attachments/assets/ec986a7c-0e4c-4af0-8da3-80a2e8118c04)

![image](https://github.com/user-attachments/assets/2a22a314-033f-404d-9db1-66f80cc26e36)

**Reply by cpcopy** [reply]
Thanks @DHowett. You are right. The issue only happens on my new win 11 laptop. No matter 1.20.11381.0. or 1.20.11781.0. issue can be reproduced. Maybe it is due to different win 11 on this laptop. But I don't find any functions bounded to ctrl+shift+L. Following is my system information.
![image](https://github.com/user-attachments/assets/78eb5e59-58dd-4c03-95d4-aaaaa0b48a2a)

**Comment by cpcopy** [comment, marked as answer]
Finally, I find the shortcut is occupied by AMDRSSERV. Delete the shortcut map from AMDRSSERV. Issue is resolved.

**Reply by DHowett** [reply]
Great find! Thank you.
